api/scripts/calculate_percentile_offline.py

```diff
@@ -127,11 +132,11 @@ def remove_data_outside_fire_season(df, season: dict):
     df.drop(df[after_end].index, inplace=True)
 
 
-def calculate_percentile(df, percentile: float) -> dict:
+def calculate_percentile(df: pd.DataFrame, percentile: float) -> dict:
     """ Calculate percentile """
-    ffmc = df[df['ffmc_valid']].ffmc.quantile(percentile)
-    bui = df[df['bui_valid']].bui.quantile(percentile)
-    isi = df[df['isi_valid']].isi.quantile(percentile)
+    ffmc = df[df['FINE_FUEL_MOISTURE_CODE'].notnull()]['FINE_FUEL_MOISTURE_CODE'].quantile(percentile)
```

Comment:
Do we now have records with missing indices?

Reply:
Could you elaborate on that?

Reply:
Got it, thanks!